frontend/public/components/dashboard/project-dashboard/inventory-card.tsx

```diff
@@ -43,27 +45,35 @@ const ProjectInventoryItem = withDashboardResources(
     mapper,
     useAbbr,
   }: ProjectInventoryItemProps) => {
+    const canListResource = useAccessReview({
```

Comment:
Hm, I can't decide if we should check access reviews for all resources. Most of the time, it's not needed. Secrets are a special case because project viewers can't see them.

Reply:
If you think that checking Secrets is enough, them Im good with it too.